test_pstreams.cc

```diff
@@ -827,8 +827,8 @@ int main()
         class pguard
         {
         public:
-            explicit pguard(ipstream& in, int signal=SIGKILL)
-            : buf_(*in.rdbuf()), signal_(signal) { }
+            explicit pguard(ipstream& is, int signal=SIGKILL)
+            : buf_(*is.rdbuf()), signal_(signal) { }
```

Comment:
Sigh, this is a stupid warning from Clang. The code is fine. I'll merge this though, just to shut the warning up.